build.gradle.kts

```diff
@@ -13,8 +13,14 @@ plugins {
     signing
     `maven-publish`
     idea
-    id("com.diffplug.gradle.spotless") version "3.25.0"
-    id("com.gradle.plugin-publish") version "0.12.0"
+
+    if (org.gradle.util.GradleVersion.current() < org.gradle.util.GradleVersion.version("5.4")) {
```

Comment:
@rnc Why is the fully qualified name required here? If I try `GradleVersion.current()` it says not found. It would make these lines a bit easier to read.

Reply:
Hmm, not sure I noticed this before and sort of ignored it :-)

Reply:
And the IDE is suggesting to remove it, but if I do, it breaks. I see it working in other examples, so I don't understand.

Reply:
Frankly, not worth bothering about in _this_ PR ; can always investigate another time IMHO. I'm merging this as all checks have now passed.